### ISSUE TITLE
Anaconda environment name with spaces causes issues

### ISSUE THREAD

**Issue by jimmyland22** [AutoViML/deep_autoviml#4]
Running deepauto.fit, I'm seeing some issues relating to my environment name having spaces in them. Example:

#################################################################################
#################################################################################
        
Creating a keras Function model...
    number of outputs = 1, output_activation = sigmoid
    loss function: SparseCategoricalCrossentropy
    initial learning rate = 0.05
    initial optimizer = SGD
    Recommended hidden layers (with units in each Dense Layer)  = (96, 64, 32)

    creating auto model body...
"dot" with args ['-Tps', 'C:\\Users\\JIMMY~1.LIA\\AppData\\Local\\Temp\\1\\tmp53d9l6vd'] returned code: 1

stdout, stderr:
 b''
b"'C:\\Users\\jimmy.liang\\Anaconda3\\envs\\Special' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n"

Model plot not saved due to error. Continuing...

**Comment by AutoViML** [comment]
Please avoid using Anaconda environment names with spaces since I am saving the saved models locally.
